### ISSUE TITLE
Multiple errors of the same kind

### ISSUE THREAD

**Issue by Quantum00** [hsoft/dupeguru#449]
There is a whole lot of this error popping up as soon as a normal scan is finished

Application Name: dupeGuru
Version: 4.0.3

Traceback (most recent call last):
  File "C:\Users\arsen\github\dupeguru\hscommon\gui\progress_window.py", line 96, in pulse
  File "C:\Users\arsen\github\dupeguru\qtlib\progress_window.py", line 44, in close
AttributeError: 'ProgressWindow' object has no attribute '_timer'

It apperas: C:\Users\arsen\ is hardcoded? There is no such directory on my pc.
Using Windows 10.

**Comment by arsenetar** [comment]
The paths reported in the Traceback are based on the location of the files before they were packaged via cx_Freeze.  Those files all are stored as .pyc files in the python35.zip that is distributed with the application. I have not found a way to change that output, "C:\Users\arsen\github\dupeguru\" is the base path in this case which the following path exists in the zip file at the installed location (as .pyc not .py).  

I have run both the x64 and x32 versions on multiple computers (all x64) under both windows 10 and 7.  Did you install it "for all users" or just for "yourself"?  Also are you using the x64 or x32 version? 

If I understand this correctly, to reproduce:

1. Launch dupeGuru
2. Select a folder and initiate a scan (progress window pops up)
3. Scan completes (progress window closes)
4. Error pops up (once per scan? or is this repeated several times?)

I currently can not reproduce it on my machines here, but looking into what could cause it.

**Comment by Quantum00** [comment]
It is x64, i think i installed it for single user.
You are right at the steps to reproduce it. Scan completes, progress window closes, results window opens and on top of that the error window pops up a whole lot of times (probably as many as the matches? I had 1500 results). I didn't count how many because they wouldn't stop so I ended dupeguru from task manager.
Trying now with a reinstall of the x64 for all users.

**Comment by arsenetar** [comment]
I managed to reproduce the issue when running with debugging, looking into a fix.

**Comment by Quantum00** [comment]
Many thanks!

After a reinstall of x64 but all users this time the bug did not occur, but do fix it for single user if you can.

**Comment by arsenetar** [comment]
That is interesting that you had a difference there, from what I found it seems there was an issue with calling close on the progress_window dialog multiple times. The issue does not seem to come up every time when I was testing.  
@hsoft I have not figured out why close() gets called multiple times but I did create hsoft/qtlib/#2 in which close() can now be called multiple times per show() without throwing an exception (later calls do nothing).
